specviz/plugins/smoothing/smoothing_dialog.py

```diff
@@ -174,7 +174,7 @@ def accept(self):
         self.smooth_button.setEnabled(False)
         self.cancel_button.setEnabled(False)
 
-        self.size = float(self.size_input.text())
+        self.size = int(self.size_input.text())
```

Comment:
I wonder if an error/warning might be better here or a check to see `int()` is close to desired.  if the person enters `3.95` then this will truncate the value to `3` which is quite a ways off.

Reply:
The field validator on the line edit will ensure the user cannot enter a float.